fast64_internal/f3d/f3d_material.py

```diff
@@ -3596,6 +3596,31 @@ def key(self):
         return str(self.to_dict().items())
 
 
+def draw_rdp_world_defaults(layout: UILayout, scene: Scene):
+    world = scene.world
+    if not world:
```

Comment:
same comment about the condition stuff as earlier (hopefully this comment comes after the one I'm talking about lol)